CalcWithMethods.java

```diff
@@ -0,0 +1,61 @@
+import java.util.Scanner;
+public class CalcWithMethods {
+
+    public static void main(String args[]) {
+
+        int num1 = getInt1();
+        int num2 = getInt2();
+        char operation = getOperation();
+        int result = calc(num1, num2, operation);
+        System.out.println("Result is: " + result);
+    }
+
+    public static int getInt1() {
+        Scanner scanner = new Scanner(System.in);
+
+        int num1;
+        System.out.print("Enter first number: ");
+        num1 = scanner.nextInt();
+
+        return num1;
+    }
+
+    public static int getInt2(){
+        Scanner scanner = new Scanner(System.in);
+
+        int num2;
+        System.out.print("Enter second number: ");
+        num2 = scanner.nextInt();
+
+        return num2;
+    }
+
+    public static char getOperation(){
+        Scanner scanner = new Scanner(System.in);

```

Comment:
Take a look on your strings 34 and 24 you have initialization of the object and usage of the same class, this is called code repeatability. So, in any of programming language there is a DRY principle. It's a bit complicated thing, but the beyond line is "Don't repeat yourself".

In your case you should initialize your object above main method, so in this case you will get only one object for both methods.